backend/src/main/java/ai/verta/modeldb/versioning/RepositoryDAORdbImpl.java

```diff
@@ -78,7 +78,7 @@
 
   private static final String SHORT_NAME = "repo";
 
-  private static final StringBuilder GET_REPOSITORY_COUNT_BY_NAME_PREFIX_HQL =
+  private static final String GET_REPOSITORY_COUNT_BY_NAME_PREFIX_HQL =
```

Comment:
the actual cause of duplicate repos and dataset and a great candidate for OOM.
We were appending to the static stringBuilder